specs/plugin-extensibility.md

```diff
@@ -375,6 +375,9 @@ All request attributes are required.
   // Optional plugin configuration, map of string-string
   "pluginConfig" : { },
 
+  // Optional signature expiry duration in seconds, number
+  "expiryDurationInSeconds" : 1234567,
```

Comment:
In addition, the `ArtifactReference` is not passed to the plugin. How would the trust policy in the plugin know which policy to apply?

Reply:
This is for sign operation not verify.

Reply:
Oh. I mixed the paths.